lms/envs/common.py

```diff
@@ -1257,7 +1257,7 @@
     'js/vendor/url.min.js',
     'common/js/vendor/underscore.js',
     'common/js/vendor/underscore.string.js',
-    'common/js/vendor/picturefill.min.js',
```

Comment:
Ah.. Thanks for that info. Didn't know and make sense. That means everywhere we use a third party js library, we should always use the un-minified version huh